src/traversal/printRelayOSSQuery.js

```diff
@@ -246,7 +246,7 @@ function printDirectives(node) {
   return ' ' + directiveStrings.join(' ');
 }
 
-function printDirective({name,value}) {
+function printDirective({ name, value }) {
```

Comment:
We actually tend to prefer this without the space padding the braces (there's another case where you added that)